src/tests/test_protectAllClasses.py

```diff
@@ -0,0 +1,20 @@
+
+from Basilisk.moduleTemplates import cModuleTemplate
```

Comment:
Could you move this test to a local test folder:
```
/src/architecture/_GeneralModuleFiles/_UnitTest
```

Reply:
Never mind that request, we already have a bunch of generic system functionality testing in `src/tests` folder.  Let's leave the script there for now.